src/__tests__/remark.js

```diff
@@ -0,0 +1,25 @@
+import remark from 'remark';
+import remarkEmbedder from '../';
+
+test('works with remark directly', async () => {
+  const result = await remark()
+    .use(remarkEmbedder)
+    .process('https://www.youtube.com/watch?v=dQw4w9WgXcQ');
+
+  expect(result.toString()).toMatchInlineSnapshot(`
+    <iframe width="100%" height="315" src="https://www.youtube-nocookie.com/embed/dQw4w9WgXcQ?rel=0" frameBorder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowFullScreen></iframe>
+  `);
```

Comment:
This assertion is failing:

```
 FAIL  src/__tests__/remark.js
  ✕ works with remark directly (16 ms)
  ✓ can pass options (2 ms)

  ● works with remark directly

    expect(received).toMatchInlineSnapshot(snapshot)

    Snapshot name: `works with remark directly 1`

    - Snapshot  - 1
    + Received  + 2

    - <iframe width="100%" height="315" src="https://www.youtube-nocookie.com/embed/dQw4w9WgXcQ?rel=0" frameBorder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowFullScreen></iframe>
    + https://www.youtube.com/watch?v=dQw4w9WgXcQ
    +

       7 |     .process('https://www.youtube.com/watch?v=dQw4w9WgXcQ');
       8 | 
    >  9 |   expect(result.toString()).toMatchInlineSnapshot(`
         |                             ^
      10 |     <iframe width="100%" height="315" src="https://www.youtube-nocookie.com/embed/dQw4w9WgXcQ?rel=0" frameBorder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowFullScreen></iframe>
      11 |   `);
      12 | });

      at Object.<anonymous> (src/__tests__/remark.js:9:29)

 › 1 snapshot failed.
Snapshot Summary
 › 1 snapshot failed from 1 test suite. Inspect your code changes or press `u` to update them.

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 passed, 2 total
Snapshots:   1 failed, 1 passed, 2 total
Time:        0.961 s, estimated 4 s
Ran all test suites matching /src\/__tests__\/remark\.js/i.
```

@wooorm, do you have any ideas? I think it may be related to the recent changes we made to use ASTs rather than a string of HTML.

Reply:
Nevermind! I just realized that I wasn't using `remark-html` which is responsible for actually taking the AST and turning it into a string of HTML. Once I added remark-html, the test passed.